teach/templates/clubs/email/create_club_staff/body.txt

```diff
@@ -0,0 +1,14 @@
+Club captain {{ full_name }} would like to add their Club to the map (owned by {{ username }}).
```

Comment:
I'm honestly going to defer to @KGorr and/or @jvallera for this copy since she will be the one receiving this email.

@KGorr / @jvallera, this is the template for the email that will be sent to you when someone creates a new club using the form. What would you prefer the first sentence of the email to be (so that it's future proof as well). 
Note that `{{ full_name }}` will be replaced with the name of the applicant as entered into the application form on the learning site, `{{ username }}` will be the Webmaker user who was logged in while submitting that form on the learning site.